### PR TITLE
OSDOCS-12016: update attributes and updates content

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.18+

Issue:
[OSDOCS-12016](https://issues.redhat.com/browse/OSDOCS-12016)

Link to docs preview:
[Compatibility table](https://83076--ocpdocs-pr.netlify.app/microshift/latest/microshift_install_get_ready/microshift-install-get-ready#get-ready-install-rhde-compatibility-table_microshift-install-get-ready)
[Removes update paths snippet as it is is duplicated content](https://83076--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-options#microshift-update-options-standalone-updates_microshift-update-options)
[Version bump in text ostree](https://83076--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-rpms-ostree)
[Version bump in text rpms](https://83076--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-rpms-manually)
[Troubleshooting updates versions](https://83076--ocpdocs-pr.netlify.app/microshift/latest/microshift_troubleshooting/microshift-troubleshoot-updates#microshift-update-path-blocked-by-version-sequence_microshift-troubleshoot-updates)

QE review:
- N/A internal docs work

### CODE REVIEW

**Review by DanielFroehlich — APPROVED** [review]
/LGTM

File: modules/microshift-updates-troubleshooting.adoc [hunk 1]
```diff
@@ -8,13 +8,9 @@
...
-//[id="microshift-update-path-blocked-by-version-sequence_{context}"]
-//== Update path is blocked by {microshift-short} version sequence
-//Certain versions of {microshift-short} require serial updates. Attempting to update {microshift-short} by skipping a minor version fails:
-
-//* For example, if your current version is `4.14.5`, but you try to update from that version to `4.16.0`, the message, `executable (4.16.0) is too recent compared to existing data (4.14.5): version difference is 2, maximum allowed difference is 1` appears and {microshift-short} fails to start.
-
-//In this example, you must first update `4.14.5` to a version of `4.15`, and then you can upgrade to `4.16.0`.
+[id="microshift-update-path-blocked-by-version-sequence_{context}"]
+== Update path is blocked by {microshift-short} version sequence
+Non-EUS versions versions of {microshift-short} require serial updates. For example, if you attempt to update from {microshift-short} `4.15.5` directly to `4.17.1`, the update fails. You must first update `4.15.5` to `4.16.z`, and then you can update from `4.16.z` to `4.17.0`.
```
Comment (tmalove) [inline modules/microshift-updates-troubleshooting.adoc:13]:
```suggestion
Non-EUS versions of {microshift-short} require serial updates. For example, if you attempt to update from {microshift-short} `4.15.5` directly to `4.17.1`, the update fails. You must first update `4.15.5` to `4.16.z`, and then you can update from `4.16.z` to `4.17.0`.
```

**Review by tmalove — APPROVED** [review]

**Pull request merged**